src/ensembl/src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import classNames from 'classnames';
+import { scaleLinear, ScaleLinear } from 'd3';
+
+import { getFeatureCoordinates } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+import { Exon } from 'src/content/app/entity-viewer/types/exon';
+import { CDS } from 'src/content/app/entity-viewer/types/cds';
+
+import styles from './UnsplicedTranscript.scss';
+
+export type Props = {
```

Comment:
Since we are exporting this, would it be better to rename this to `UnsplicedTranscriptProps` ?

Reply:
✔️ 

turned out these props aren't imported anywhere; so technically they don't even need to be exported; but I suppose it wouldn't hurt.